patches/third_party/catapult/common-py_utils-py_utils-cloud_storage.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/common/py_utils/py_utils/cloud_storage.py b/common/py_utils/py_utils/cloud_storage.py
+index b9f155e18b1fbf32e0888ea8f2e9e1a722cb5fa5..8c02713af45b81a0d7d8f858b1c13272a8fa76a7 100644
+--- a/common/py_utils/py_utils/cloud_storage.py
++++ b/common/py_utils/py_utils/cloud_storage.py
+@@ -30,7 +30,7 @@ logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+ 
+ 
+ PUBLIC_BUCKET = 'chromium-telemetry'
+-PARTNER_BUCKET = 'chrome-partner-telemetry'
```

Comment:
is this strictly necessary? Do we really care what the bucket name is?

Reply:
Yep, we can't download our files from Google bucket.
They are different from Google ones. 

Also, `chrome-partner-telemetry` is restricted.

Reply:
but aren't we using a different url or are these global?

Reply:
This is a global things, which can be used in many tests.
The sense of the patch is to redirect all these tests to our bucket.